downloads/index.md

```diff
@@ -348,7 +348,7 @@ The platforms currently supported by Julia are listed below. They are divided in
       <td rowspan="2"> FreeBSD </td>
       <td rowspan="2"> 11.0+ </td>
       <td> x86-64 (64-bit) </td>
-      <td> <font color="green">Tier 1</font> </td>
+      <td> <font color="green">Tier 2</font> </td>
```

Comment:
```suggestion
      <td> <font color="green">Tier 1</font> </td>
```
This was addressed by https://github.com/JuliaLang/www.julialang.org/pull/1345.

Reply:
This isn't resolved

Reply:
This is still updating FreeBSD to  be tier 2, as intended. Although I think (after this PR), it was mentioned we should also add mention of these changes to the current release notes?